mms/serving_frontend.py

```diff
@@ -40,10 +40,23 @@ def __init__(self, app_name):
         app_name : str
             App name to initialize request handler.
         """
+        def gen_prefix(prefix, length=5, chars=string.ascii_letters+string.digits):
+            return prefix + ''.join(random.choice(chars) for _ in range(length)) + '_'
+
         try:
             self.service_manager = ServiceManager()
             self.handler = FlaskRequestHandler(app_name)
 
+            if batching:
+                # generate a pseudo-random prefix to avoid collisions between runs
+                prefix = gen_prefix(app_name)
```

Comment:
Wouldn't this tie each front-end to its own model-services? You might want to consider removing this if you want other model-service threads to handle the queues.

Reply:
fixed